swift/apple/FirezoneKit/Sources/FirezoneKit/Views/macOSAlert.swift

```diff
@@ -12,25 +12,27 @@ import NetworkExtension
 
 @MainActor
 struct macOSAlert { // swiftlint:disable:this type_name
-  static func show(for error: Error) {
+  static func show(for error: Error) async {
     if let error = error as? OSSystemExtensionError,
-       // Expected in normal operation
-       error.code != .requestCanceled,
-       error.code != .requestSuperseded,
-       error.code != .authorizationRequired {
-      alert(message(for: error))
+       [
+        // Expected in normal operation
+        .requestCanceled,
+        .requestSuperseded,
+        .authorizationRequired
+       ].contains(error.code) {
+      return
```

Comment:
This control-flow isn't something I'd expect in a helper class like this. Perhaps we can move this one layer up? Or introduce a "handle_error" function that contains this plus the Alert.show code?

Reply:
Hm yeah, what's probably clearest is for this alert function to just accept an ignoringErrors array so the caller can pass them depending on the context.

Reply:
Actually that won't work because Errors need to be typed before we can act on their code (code numbers can overlap).

I'll update the message function to return nil for errors we don't want to show the user, and then simply no-op the modal alert if the message is nil.